frontend/src/components/RecipeDraft.js

```diff
@@ -65,34 +96,19 @@ export default function RecipeDraft({recipe, onRecipeNameUpdated, onIngredientsU
                         onDelete={onDeleteIngredient}
                     />
                 ))}
-
-                    <IconButton className={classes.btnLeft} type="button" onClick={onAddIngredient}>
-                        <AddCircleOutlineOutlinedIcon/>
-                    </IconButton>
-                    <Button className={classes.btnRight} type="submit"><CheckOutlined/></Button>
-
+                <IconButton className={classes.btnLeft} type="button" onClick={onAddIngredient}>
+                    <AddCircleOutlineOutlinedIcon/>
+                </IconButton>
+                <Button className={classes.btnRight} type="submit">
+                    <CheckOutlined/>
+                </Button>
             </form>
-        </>
+        </MuiThemeProvider>
     )
 }
 
 function uuid() {
```

Comment:
better use a battle tested package for this `npm install uuid` 😉

Reply:
I'll stick with this one for now, apparently the "default" npm uuid can provide doubles when you are past 50 million IDs (not that I will ever reach that) and this is a "fix" for that, also I have read far too much about uuid structure while researching this, so I don't want to change it when there is no problem with it. :)

Reply:
Okay, i suggest refactoring it to a service then, so it is less painful should you decide to change it later.